app/models/abstract_model.rb

```diff
@@ -413,9 +415,9 @@ def show_action
   # Note that show_controller has a leading forward slash
```

Comment:
Revise documentation (`show_controller` is now a symbol without a leading forward slash)